src/Compilers/VisualBasic/Portable/Symbols/NamedTypeSymbol.vb

```diff
@@ -192,6 +192,18 @@ Namespace Microsoft.CodeAnalysis.VisualBasic.Symbols
         ''' </summary>
         Public MustOverride ReadOnly Property IsSerializable As Boolean Implements INamedTypeSymbol.IsSerializable
 
+        Public ReadOnly Property IsNativeInteger As Boolean Implements INamedTypeSymbol.IsNativeInteger
```

Comment:
>IsNativeInteger [](start = 33, length = 15)

Please follow the convention to prefix API names with interface names, see implementations for other INamedTypeSymbol APIs below. Also, consider moving these methods next to them.